cohort-engine-api-web/src/main/java/com/ibm/cohort/engine/api/service/CohortEngineRestHandler.java

```diff
@@ -386,6 +386,7 @@ public Response getMeasureParametersById(@Context HttpHeaders httpHeaders,
 	}
 
 	public final static String VALUE_SET_PART = "valueSet";
+	public final static String CUSTOM_CODE_SYSTEM = "customCodeSystem";
```

Comment:
@tabishop has [a PR](https://github.com/Alvearie/quality-measure-and-cohort-service/pull/104) open right now for updating parameters names to use underscores instead of camel case. Not sure which of you will come down first, but we need to make sure that we line these two changes up to be consistent.

Reply:
Correct, we are going with lower_snake_case for input parameters to line up with the rest-service-framework so this should be custom_code_system to align.

Reply:
Alright, I can either make that change now or wait for the change in Tim's PR, but there doesn't feel like a consensus in this comment thread.

Reply:
I think Corey and I are in agreement so I would like it if you made the change in your PR for this parameter to be custom_code_system. Once my PR is merged, the input parameters will all be lower_snake_case. Mine is just waiting for another review before it goes in.

Reply:
alright, pushed.